src/Units/WeaponModel.java

```diff
@@ -6,15 +6,22 @@
 import Units.AWBWUnits.AWBWUnitModel;
 import Units.DoRUnits.DoRUnitModel;
 import Units.KaijuWarsUnits.KaijuWarsUnitModel;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.Accessors;
 
+@Accessors(fluent = true)
+@Getter(AccessLevel.PUBLIC)
```

Comment:
Isn't PUBLIC the default?

Reply:
[This](https://projectlombok.org/features/GetterSetter) seems to confirm that, but extra clarity also doesn't hurt.